packages/katello/katello/hostname-change.rb

```diff
@@ -345,6 +345,53 @@ def run
       STDOUT.puts "updating hostname in foreman installer scenarios"
       self.run_cmd("sed -i -e 's/#{@old_hostname}/#{@new_hostname}/g' #{scenarios_path}/*.yaml")
 
+      if @scenario_answers['foreman_proxy']['dns']
```

Comment:
this file isn't the best about this, but it does seem like this could be refactored into method(s) called here for readability. maybe even 3 or more methods. My first instinct is to separate into: `get_dns_data` , `forward_dns_command`, `reverse_dns_command`, and `update_dns_records`,  or even have one `dns_command` that takes a large string to echo into nsupdate and create additional methods returning those strings for forward and reverse (in heredoc or %Q{foo} to split into multiple lines like ewoud suggested). 

Hopefully this make sense!

Reply:
@johnpmitsch thanks, updated!  Couldn't figure out an elegant way to pass data to `forward_dns_command` and `reverse_dns_command`, so keeping them together in `update_dns_records` for now.  (If only Ruby had JS ES6 destructuring ;)

And I had to use `<<-` heredocs because Ruby 2.0 doesn't support `<<~` :cry: